plugin/trino-oracle/src/test/java/io/trino/plugin/oracle/TestingOracleServer.java

```diff
@@ -123,4 +125,14 @@ public void close()
     {
         stop();
     }
+
+    public static Map<String, String> connectionProperties(TestingOracleServer server)
```

Comment:
That belongs to the query runnner class.

_This_ class is not concerned with Trino connector or its configuration